packages/react-components/src/templates/TeamCreateOutputPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ResearchOutput } from '@asap-hub/model';
+
+import { TeamCreateOutputHeader, TeamCreateOutputForm } from '../organisms';
+
+type TeamCreateOutputPageProps = {
+  onCreate: () => void;
+  researchOutput: ResearchOutput;
+};
+
+const TeamCreateOutputPage: React.FC<TeamCreateOutputPageProps> = ({
+  researchOutput,
+  onCreate,
+}) => (
+  <>
+    <TeamCreateOutputHeader type={researchOutput.type} />
```

Comment:
> Generally we pass in the entire research output like {...researchOutput} this means we don't have to remap as many props and sometimes we can get away with requiring additional attributes

Reply:
> I was hoping we could refactor the modal save functionality into something generic we could use on this form. It does things like deal with disabling inputs and buttons while we are saving and asking the users if they are sure they want to navigate away without saving changes

![image](https://user-images.githubusercontent.com/6905396/151000842-fb7c1b3e-d974-41ad-b7dd-6c5ca8d1051d.png)

Reply:
I think this is part of another ticket coming up soon.

Reply:
Not sure you have replied to the comment above

Reply:
Think the above belongs in another comment